pocs/camera/sbig.py

```diff
@@ -190,11 +210,21 @@ def take_exposure(self, seconds=1.0 * u.second, filename=None, dark=False, block
 
         assert filename is not None, self.logger.warning("Must pass filename for take_exposure")
 
-        self.logger.debug(
-            'Taking {} second exposure on {}: {}'.format(
-                seconds, self.name, filename))
+        if self.focuser:
```

Comment:
This should be integrated in a more generic fashion. For example, the base focuser could have a get_fits_headers method.

Reply:
Yes. Reorganisation of the FITS header related code has been on the todo list for a while: https://github.com/AstroHuntsman/HuntsmanPOCSLegacy/issues/28